docs/docs-source/docs/modules/get-started/pages/install-cloudflow.adoc

```diff
@@ -0,0 +1,322 @@
+= Installing Cloudflow
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+This guide shows how to install Cloudflow and dependencies on Cloudflow like the Spark and Flink operator, using the `Helm` and `kubectl` command-line tools. 
+
+== Prerequisites
+
+=== CLI Tools
+
+Make sure you have the following prerequisites installed before continuing:
+
+- Helm, version 3 or later
+- Kubectl 
+
+Before proceeding, make sure that `kubectl` is correctly installed and access the Kubernetes cluster to install Cloudflow.
+
+=== Kafka
+
+Kafka is used by Cloudflow to store information. Before installing Cloudflow, you need to make sure there is a Kafka cluster available and have the necessary broker bootstrap configuration.
+
+The Kafka broker bootstrap configuration string is a comma-separated list of host/port pairs used by Cloudflow to establish the connection to the Kafka cluster. The configuration string should have the following format:
+
+broker-1-address:broker-1-port, broker-2-address;broker-2-port
+
+If you want to test Cloudflow and need a Kafka cluster, we recommend using Strimzi, a third-party Kafka operator that can create and manage Kafka clusters. 
+
+The following chapter is a guide on how to configure and install a Kafka cluster using Strimzi. 
+
+. xref:install-and-use-strimzi.adoc[How to install Strimzi and create a Kafka cluster]
```

Comment:
Following that, I don't see what the bootstrap servers must be when using strimzi, only in upgrade doc there is a mention of `cloudflow-strimzi-kafka-bootstrap.cloudflow:9092`

Reply:
Oh wait that is showed later, never mind!